### ISSUE TITLE
Upload command does not add dotfiles

### ISSUE THREAD

**Issue by achilleas-k** [G-Node/gin-cli#51]
This is a tricky one.
*Should* we add dotfiles?

I might rethink this if I add a command that does `git add` exclusively.
Also under consideration is a command that performs `git add $* && git commit ...`, to record changes locally without necessarily uploading.

**Comment by cgars** [comment]
DOI can be (actually has been) changed to cloudberry instead of .cloudberry. Needs to be done in UI too, i guess. 

In general i think cli should be able to do it (whether it should do it by default is another question)

**Comment by achilleas-k** [comment]
I agree that it should be able to do it. Still not sure whether it should do it by default.
I suppose if we're thinking in the "directory sync" sense, we should be syncing dotfiles and basically everything.
